File: js210/js210_small_problems/easy_5/reverse_it_2.js
"use strict"

function reverseWords(string) {
  let result = [];
  let words = string.split(' ');

  for (let i = 0; i < words.length; i += 1) {
    let word = words[i];

    if (word.length >= 5) {
      result.push(word.split('').reverse().join(''));
    } else {
      result.push(word);
    }
  }

  return result.join(' ');
}

console.log(reverseWords('Professional'));
console.log(reverseWords('Walk around the block'));
console.log(reverseWords('Launch School'));

File: rb109/Practice/Easy 1/reverse_it_2.rb
# reverse_it_2.rb

def reverse_words(string)
  words = string.split

  reversed_words = words.map do |word|
    if word.length >= 5
      word.chars.reverse.join
    else
      word
    end
  end

  reversed_words.join(' ')
end

puts reverse_words('Professional')
puts reverse_words('Walk around the block')
puts reverse_words('Launch School') 
